generator/insn_vdvs2rs1vm.go

```diff
@@ -9,12 +9,15 @@ import (
 func (i *Insn) genCodeVdVs2Rs1Vm(pos int) []string {
 	vdWidening := strings.HasPrefix(i.Name, "vw")
 	vs2Widening := strings.HasSuffix(i.Name, ".wx")
+	sew64_insn := i.Name == "vclmul.vx" || i.Name == "vclmulh.vx"
 	vdSize := iff(vdWidening, 2, 1)
 	vs2Size := iff(vs2Widening, 2, 1)
 
 	sews := iff(vdWidening || vs2Widening, allSEWs[:len(allSEWs)-1], allSEWs)
+	sews = iff(sew64_insn, []SEW{64}, sews)
+
 	combinations := i.combinations(
-		iff(vdWidening || vs2Widening, wideningMULs, allLMULs),
+		iff(vdWidening || vs2Widening, wideningMULs, iff(sew64_insn, []LMUL{1, 2, 4, 8}, allLMULs)),
```

Comment:
So, IIUC, for like VLEN=128, LMUL=1/2 should be okay?

Reply:
spike returned illegal instruction trap for all fractional LMUL

Reply:
Okay, but can you explain why?

Reply:
Chapter 4, Section 2 of the riscv vector spec specifies that LMUL can not be less then 1 when SEW=64

Reply:
<img width="1372" alt="image" src="https://github.com/user-attachments/assets/e4b154eb-96cc-4886-ad04-2413c4ad1eaa">
Are you sure? I see no similar statements in 4.2.